soh/src/overlays/actors/ovl_En_Syateki_Man/z_en_syateki_man.c

```diff
@@ -154,6 +154,15 @@ void EnSyatekiMan_Init(Actor* thisx, GlobalContext* globalCtx) {
     s32 pad;
     EnSyatekiMan* this = (EnSyatekiMan*)thisx;
 
+    if (gSaveContext.n64ddFlag) {
```

Comment:
do we want to do this always in rando? or just when doing entrance rando?

Reply:
3ds rando doesn't do a check for this, but I can add one here (for interior shuffle) if we want.

Reply:
Are there any glitches that involve child link going to adult shooting gallery or vice versa?

If so we probably should add an ER check here, if not this is probably fine.

Reply:
I'm not sure the usefulness, but I did find an example of child linking clipping into adult shooting gallery. I added an interior shuffle check here now.